gslab_scons/tests/test_release_function.py

```diff
@@ -80,133 +144,103 @@ def test_release_standard(self, mock_move, mock_make_archive, mock_copy,
         # of session.get()
         self.assertEqual(mock_upload.call_args[1]['release_id'], 'test_ID')
 
-        # Check that release() zipped the files released to Google Drive correctly...
-        mock_copy.assert_any_call('paper.pdf', 'release_content/paper.pdf')
-        mock_copy.assert_any_call('plot.pdf',  'release_content/plot.pdf')
-        mock_make_archive.assert_called_with('release_content', 'zip', 'release_content')
-        # ...and moved them to the local_release Google Drive directory
-        mock_move.assert_called_with('release_content.zip', 
-                                     'Google Drive/release/local_release/release.zip')
+        # Check that release() prepared files for release correctly,
+        # If we are zipping before releasing, then we should move
+        # our release files into an intermediate folder to be zipped.
+        if args['zip_release']:
+            base = 'release_content'
+        # Otherwise, we should move them directly into our 
+        # local_release directory.
+        else:
+            base = args['local_release']
+
+        drive_files = args['DriveReleaseFiles']
+        if isinstance(drive_files, basestring):
+            drive_files = [drive_files]
+
+        for filename in drive_files:
+            mock_copy.assert_any_call(filename, '%s/%s' % (base, filename))
```

Comment:
`assert_called_with()` considers only the most recent call of the mocked function, so it would fail if `len(drive_files) > 2`. See [here](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_any_call).